source/client/model/Cube.cpp

```diff
@@ -51,6 +51,18 @@ Cube::Cube(ModelPart* a2, int a3, int a4, float x, float y, float z, int d, int
 	m_faces[4] = PolygonQuad(&m_verts[1], &m_verts[0], &m_verts[3], &m_verts[2], m + f,         n + f, m + f + d,         n + f + e);     // z1 face
 	m_faces[5] = PolygonQuad(&m_verts[4], &m_verts[5], &m_verts[6], &m_verts[7], m + f + d + f, n + f, m + f + d + f + d, n + f + e);     // z2 face
 
+#ifdef ENH_ENTITY_SHADING && 0
```

Comment:
What does this do?

Reply:
I was messing around with doing per-face shading on entities, like what's done manually with in-hand blocks. I didn't wanna completely trash the work, so I just didn't remove it and instead #ifdefed it out.